### PR TITLE
Toast

### PR DESCRIPTION
**Pull request by AshutoshM29**
Preview : https://deploy-preview-10--uiclean.netlify.app/pages/toast/toast

### CODE REVIEW

File: Pages/Toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+.toast {
+    display: flex;
+    flex-direction: row;
+    background-color: var(--gray-lighter);
+    /* color: var(--blue-dominant); */
+    width: fit-content;
+    align-items: center;
+    padding: 0 1.5rem 0 1.5rem;
+    border-radius: 0.5rem;
```
Comment (Sahil25061999) [inline Pages/Toast/toast.css:9]: you can just use padding: 0 1.5rem here

File: Pages/Toast/toast.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+.toast-container {
+    border: solid 1rem var(--blue-dominant);
+    display: flex;
+    flex-direction: column;
+    padding: 1.6rem;
+    margin: 3rem;
+    max-width: 80%;
+    border: 0.5rem solid var(--blue-dominant);
+    border-radius: 1rem;
+    display: flex;
```
Comment (Sahil25061999) [inline Pages/Toast/toast.css:21]: display: flex is repeated

**Review by Sahil25061999 — COMMENTED** [review]
Can be merged with minor changes

**Comment by manish-GH** [comment]
Toast looks good but you seemed to have forgotten to add the code snippet

**Comment by kgajriaAgarwal** [comment]
i suppose this is not responsive as it is getting distorted on trying responsiveness.

**Pull request merged**